app_server/app/Router.js

```diff
@@ -83,7 +85,6 @@ server.register({
   }
 });
 
-/* Register controllers */
```

Comment:
Yup, it's a duplicate. There is another one above.

Sure I can do the replacements. The `server.app.authenticator` is used to cut the circular `require`: Authenticator -> Service -> SocketAdapter -> Authenticator. By right, the cookie should be attached during the initial websocket http handshake and hence can follow the normal authentication flow before reaching the `SocketAdapter`. But that could be a bit more involved on the client so this is an adhoc way of dealing with it on the server.